### PR TITLE
Add a Cypress spec for the help and resources tab

### LINKED ISSUE #1666: Write Cypress test for Help & Resources
To test that Help and Resources can be opened and closed from the home page and each of the Sinopia tabs.

This is to prevent a regression of the bug that prompted https://github.com/LD4P/sinopia_editor/pull/1660.

I tried unsuccessfully to write this with RTL but was problematic because the offset is handled in the root component. This is more suited for a cypress test.

### PR DESCRIPTION
**Pull request by jcoyne**
Fixes #1666

### CODE REVIEW

File: cypress/integration/helpAndResources.spec.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      .type(Cypress.env('COGNITO_TEST_USER_NAME'))
+      .should('have.value', Cypress.env('COGNITO_TEST_USER_NAME'))
+    cy.get('#password')
+      .type(Cypress.env('COGNITO_TEST_USER_PASS'))
+      .should('have.value', Cypress.env('COGNITO_TEST_USER_PASS'))
+    cy.contains('Login').click()
+    cy.contains('current cognito user: sinopia-devs_client-tester')
+  })
+
+  it('Opens Linked Data Editor', () => {
```
Comment (jgreben) [inline cypress/integration/helpAndResources.spec.js]: I noticed that the issue specified that the Help and Resources menu could be opened from each of the Sinopia tabs, but this test only tests the Resource Templates tabs. @justinlittman, is this sufficient?

Comment (justinlittman) [reply cypress/integration/helpAndResources.spec.js]: Each of tabs need to be tested separately. (The bug that initiated this was only on some tabs.)

Comment (jcoyne) [reply cypress/integration/helpAndResources.spec.js]: Covering every permutation of features (thinking broader than the issue at hand) in cypress is going to create a very slow test suite.  Can we just cover the cases that caused the trouble?

Comment (justinlittman) [reply cypress/integration/helpAndResources.spec.js]: Philosophically I agree with you. However, in this case we want to make sure that a dev doesn't break the menu for any tab (as there are 2 places in code each tab can be broken, and it is not immediately obvious). So this isn't just a case of "test everything" but rather test a specific vulnerability due to the structure of the code that actually occurred (I broke some of the tabs, not all.)

**Pull request merged**